scripts/bootstrap.sh

```diff
@@ -48,17 +48,45 @@ if [ "$family" = "arch" ]; then
 		cd $WORKDIR || exit
 	fi
 	#echo "INFO: installing AUR packages"
-	
-	install_mangohud
-	install_flatpaks
-
 	echo "Install zen kernel? [y/N]"
 	read -r zen
 	if [ "$zen" == "y" ]; then sudo pacman -S linux-zen; fi
+# ubuntu/debian
+elif [ -f /usr/bin/apt ]; then
+		echo "INFO: enabling multilib "
+		dpkg --add-architecture i386
+		echo "INFO: updating system & installing packages"
+		apt update && apt upgrade
+		apt install -y  mesa-vulkan-drivers flatpak libvulkan1 vulkan-utils mesa mesa:i386 wine
+		on_ubuntu=$(lsb_release -i)
+		if [ "$on_ubuntu" == "Distributor ID: Ubuntu" ]; then
```

Comment:
Could just check the codename or `-s` but this works too.

As an side, I'd say we only support distros that ship `lsb_release` (some don't)

Reply:
I agree `lsb_release` should be shipped by the distro